src/EnergyPlus/AirflowNetworkBalanceManager.hh

```diff
@@ -249,12 +249,14 @@ namespace AirflowNetworkBalanceManager {
 
 	Real64
     CalcWindPressure(
-        int const curve, // Curve index, change this to pointer after curve refactor
-        Real64 const Vref, // Velocity at reference height
-        Real64 const height, // Node height for outdoor temperature calculation
-        Real64 const azimuth, // Azimuthal angle of surface
-        bool const symmetricCurve, // True if the curve is symmetric (0 to 180)
-        bool const relativeAngle // True if the Cp curve angle is measured relative to the surface
+		int const curve, // Curve index, change this to pointer after curve refactor
+		bool const symmetricCurve, // True if the curve is symmetric (0 to 180)
+		bool const relativeAngle, // True if the Cp curve angle is measured relative to the surface
+		Real64 const azimuth, // Azimuthal angle of surface
+		Real64 const windSpeed, // Wind velocity
+		Real64 const windDir, // Wind direction
+		Real64 const dryBulbTemp, // Air node dry bulb temperature
+		Real64 const humRat // Air node humidity ratio
```

Comment:
Is there an indentation issue here? Looks like maybe two tabs?

Reply:
I think there supposed be two tabs here, as the other functions do in this header file.